pkg/cluster/ClusterService.go

```diff
@@ -113,15 +112,15 @@ func GetClusterBean(model repository.Cluster) ClusterBean {
 	return bean
 }
 
-func (bean ClusterBean) GetClusterConfig() util.ClusterConfig {
-	return util.ClusterConfig{
+func (bean ClusterBean) GetClusterConfig() util3.ClusterConfig {
```

Comment:
almost copy of method with same name implemented on ClusterService. eliminate one.